src/utils/cosmosdb_model.ts

```diff
@@ -0,0 +1,188 @@
+// tslint:disable: member-ordering
+
+import { right } from "fp-ts/lib/Either";
```

Comment:
@pagopa/contributors migration of the base cosmos model is pretty much done, please check `cosmosdb_model` and `cosmosdb_model_versioned` - I'm going to finish writing tests and migrating the existing models